test/link/elf.zig

```diff
@@ -82,6 +82,7 @@ pub fn testAll(b: *Build, build_opts: BuildOptions) *Step {
 
         // glibc tests
         elf_step.dependOn(testAsNeeded(b, .{ .target = gnu_target }));
+        elf_step.dependOn(testLibraryPathsCompatibility(b, .{ .target = gnu_target, .use_lld = true }));
```

Comment:
Bonus points here if this test can be made to pass with `.use_lld = false` too.

Reply:
I've added an invocation with `.use_lld = false` and it also passes.